03.bowling/bowling.rb

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/ruby
+# frozen_string_literal: true
+
+class MyBowling
+  def initialize(text)
+    create_frames(text)
```

Comment:
インスタンス変数の宣言・代入はinitializeでやった方がそのクラスで使われるインスタンス変数が明確になるのでオススメです。

```suggestion
    @frames = create_frames(text)
```

Reply:
修正しました 🙏🏼 

[cd13d66](https://github.com/nagatanuen/ruby-practices/pull/3/commits/cd13d66ef1e2c1356157118d7bff2c0256032ccc)